core/queryalgebra/evaluation/src/test/java/org/eclipse/rdf4j/query/algebra/evaluation/util/ValueComparatorPerformanceTests.java

```diff
@@ -0,0 +1,42 @@
+package org.eclipse.rdf4j.query.algebra.evaluation.util;
+
+import org.eclipse.rdf4j.model.Model;
+import org.eclipse.rdf4j.model.Value;
+import org.eclipse.rdf4j.model.util.LexicalValueComparator;
+import org.eclipse.rdf4j.rio.RDFFormat;
+import org.eclipse.rdf4j.rio.Rio;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class ValueComparatorPerformanceTests {
+
+	public static void main(String[] args) throws IOException {
+
+		System.out.println("Parsing");
+		Model parse = Rio.parse(ValueComparatorPerformanceTests.class.getClassLoader().getResourceAsStream("datagovbe-valid.ttl"), "", RDFFormat.TURTLE);
```

Comment:
just checking: is that dataset something we have permission for to include in our code base?

Reply:
Yes. We did a CQ for it and use it for benchmarking in the ShaclSail.